docs/UserGuide.md

```diff
@@ -79,24 +79,28 @@ Format: `help`
 
 Adds a customer to ReadyBakey’s contact list
 
-Format: `addp n/NAME p/PHONE_NUM a/ADDRESS`
+Format: `addp n/NAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]`
 
 Examples:
 * `addp n/John Doe p/87654321 a/21 Kent Ridge Drive`
```

Comment:
Will this example work? Is email non compulsory? if so we should put it in [] like tags right?

Reply:
Good catch, removed it